packages/form-validations/README.md

```diff
@@ -0,0 +1,100 @@
+# @lmc-eu/spirit-form-validations
+
+> Form validations for Web and Web-Twig packages of the Spirit Design System.
+
+## Install
+
+```shell
+yarn add @lmc-eu/spirit-form-validations
+```
+
+or
+
+```shell
+npm install --save @lmc-eu/spirit-form-validations
+```
+
+## Usage
+
+Include the javascript file in your html head or just before the closing body tag
```

Comment:
What about?

```suggestion
Include the JavaScript file in your HTML `<head>` or just before the closing `<body>` tag
```

Reply:
Hmm, this will need usage of https://github.com/lmc-eu/code-quality-tools/tree/main/packages/textlint-rule-preset-lmc soon.

Reply:
Fixed.